javascript/chronometer.js

```diff
@@ -1,26 +1,31 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
   startClick(callback) {
-    // ... your code goes here
```

Comment:
NIce work Rich! You just forgot to assign `setInterval` to your `intervalId` but besides that everything is correct
```
this.intervalId = setInterval(() => {
      this.currentTime += 1;
    }, 1000);